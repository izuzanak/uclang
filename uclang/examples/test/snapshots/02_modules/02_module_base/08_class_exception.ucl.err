 ---------------------------------------- 
Exception: ERROR: in file: "08_class_exception.ucl" on line: 27
    try { [1,2,3][3]; } catch (except_1) {}

Index 3 exceeds array range
 ---------------------------------------- 
