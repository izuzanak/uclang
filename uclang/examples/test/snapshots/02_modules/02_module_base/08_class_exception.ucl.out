
=== Exception object ===========================================================

Exception: Exception
Exception: Exception
Exception: Exception

=== Exception method throw#0 ===================================================

Exception: Exception

=== Exception method get_type#0 ================================================

except_0.get_type() -> error_DIVISION_BY_ZERO
except_1.get_type() -> error_ARRAY_INDEX_EXCEEDS_RANGE
except_2.get_type() -> error_CLASS_DOES_NOT_CONTAIN_METHOD
except_3.get_type() -> error_USER_EXCEPTION

=== Exception method get_value#0 ===============================================

except_0.get_value() -> <blank>
except_1.get_value() -> <blank>
except_2.get_value() -> <blank>
except_3.get_value() -> My exception

=== Exception method get_file_name#0 ===========================================

except_0.get_file_name() -> 08_class_exception.ucl
except_1.get_file_name() -> 08_class_exception.ucl
except_2.get_file_name() -> 08_class_exception.ucl
except_3.get_file_name() -> 08_class_exception.ucl

=== Exception method get_line#0 ================================================

except_0.get_line() -> 24
except_1.get_line() -> 25
except_2.get_line() -> 26
except_3.get_line() -> 39

=== Exception method error_print#0 =============================================


=== Tutorial end ===============================================================

